### PR TITLE
[plugin.video.greekvoice] 4.0.3

### PR DESCRIPTION
**Pull request by threshold84**
### Description

Greekvoice is an addon featuring streams from the website wpso.com and its network located in Tampa, Florida, USA
You can stream video and radio stations found in the website and its youtube channel

### Checklist:
- [X] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [X] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [X] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.greekvoice/resources/lib/scrapetube/wrapper.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```
Comment (romanvm) [inline plugin.video.greekvoice/resources/lib/scrapetube/wrapper.py]: No need for file encoding declaration in Python 3. It's ignored.

**Review by romanvm — APPROVED** [review]
LGTM. The encoding thing is not mandatory.

**Comment by threshold84** [comment]
> LGTM. The encoding thing is not mandatory.

Thank you for approving the PR, it was about time, let me fix a couple of things, and I will wait for your merge.

File: plugin.video.greekvoice/resources/lib/scrapetube/wrapper.py (hunk 1 above)
Comment (threshold84) [reply plugin.video.greekvoice/resources/lib/scrapetube/wrapper.py]: Done

File: plugin.video.greekvoice/resources/__init__.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#  Greek Voice Add-on
```
Comment (enen92) [inline plugin.video.greekvoice/resources/__init__.py]: not needed anymore with python3

File: plugin.video.greekvoice/addon.xml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en_GB">Greek Voice addon</summary>
+        <description lang="en_GB">This addon offers live-streaming content in the greek language from the WPSO.com and XAMOSAM.com websites and its TV network located in Tampa, Florida, USA.</description>
+        <summary lang="el_GR">Πρόσθετο Greek Voice</summary>
+        <description lang="el_GR">Το πρόσθετο αυτό προσφέρει ζωντανές ροές στην ελληνική γλώσσα από τον ιστότοπο WPSO.com και XAMOSAM.com και το τηλεοπτικό του δίκτυο που βρίσκεται στην Τάμπα, Φλόριντα, ΗΠΑ.</description>
+        <language>en el</language>
+        <license>GPL-3.0, MIT Licence</license>
```
Comment (enen92) [inline plugin.video.greekvoice/addon.xml]:
```suggestion
        <license>GPL-3.0-only, MIT</license>
```

File: plugin.video.greekvoice/resources/lib/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#  Greek Voice Add-on
```
Comment (enen92) [inline plugin.video.greekvoice/resources/lib/__init__.py]: not needed in py3

File: plugin.video.greekvoice/resources/lib/runner.py [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+        {
+            'title': __lang__(30003),
+            'icon': os.path.join(ART_FOLDER, 'youtube.jpg'),
+            'fanart': FANART,
+            'kind': 'youtube'
+        }
+    ]
+
+    for m in menu:
+        list_item = xbmcgui.ListItem(m['title'])
```
Comment (enen92) [inline plugin.video.greekvoice/resources/lib/runner.py]: might consider to add offscreen=True as argument in the initializer (avoid gui locks)

File: plugin.video.greekvoice/resources/lib/runner.py [hunk 6]
```diff
@@ -0,0 +1,187 @@
...
+
+        menu = wrapper.list_channel_videos(channel_url=YT_CHANNEL_URL, limit=50)
+
+    else:
+
+        return
+
+    # noinspection PyUnboundLocalVariable
+    for m in menu:
+        list_item = xbmcgui.ListItem(m['title'])
```
Comment (enen92) [inline plugin.video.greekvoice/resources/lib/runner.py]: might consider to add offscreen=True as argument in the initializer (avoid gui locks)

**Review by enen92 — COMMENTED** [review]
Looks fine apart from the minors, should be merged after the comments have been addressed

File: plugin.video.greekvoice/resources/lib/runner.py (hunk 5 above)
Comment (threshold84) [reply plugin.video.greekvoice/resources/lib/runner.py]: Done

File: plugin.video.greekvoice/resources/lib/runner.py (hunk 6 above)
Comment (threshold84) [reply plugin.video.greekvoice/resources/lib/runner.py]: Done

**Review by enen92 — APPROVED** [review]

**Pull request merged**